src/Umbraco.Cms.Api.Common/Security/Paths.cs

```diff
@@ -14,7 +16,7 @@ public static class BackOfficeApi
 
         public static readonly string RevokeEndpoint = EndpointPath($"{EndpointTemplate}/revoke");
 
-        private static string EndpointPath(string relativePath) => $"/umbraco/management/api/v1/{relativePath}";
+        private static string EndpointPath(string relativePath) => $"/umbraco{Constants.Web.ManagementApiPath}v1/{relativePath}";
     }
```

Comment:
Maybe not include first and last slash in this constant? So you have:

```suggestion
        private static string EndpointPath(string relativePath) => $"/umbraco/{Constants.Web.ManagementApiPath}/v1/{relativePath}";
    }
```
It's more readable.

Reply:
Yea I considered that --- but then surrounding slashes have to be included everywhere the constant is used. In my opinion, all the other code changes benefit from the slashes being a part of the constant. And this one is after all a private static helper method.